src/components/Contentful/Hero.js

```diff
@@ -1,6 +1,8 @@
 import React, { Fragment } from 'react'
 import threeSpaceToLineBreak from '../../helpers/threeSpaceToLineBreak'
 import threeHyphenToSoftHyphen from '../../helpers/threeHyphenToSoftHyphen'
+import { FakeThing } from '@madetech/frontend'
```

Comment:
uhm, really?!

Reply:
I am incredibly sorry for this error, it is now deleted